### PR TITLE
[CDB-1392] Manage dashboard RBAC

### PR DESCRIPTION
**Pull request by brian-f-harness**
Manage dashboard RBAC

For reviewers, a preview is available.

[Manage dashboards](https://658376e9caaaa77200ca900b--harness-developer.netlify.app/docs/platform/role-based-access-control/manage-dashboards)

PDF - [Manage dashboards _ Harness Developer Hub.pdf](https://github.com/harness/developer-hub/files/13733480/Manage.dashboards._.Harness.Developer.Hub.pdf)

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by Rohit-Kaliki** [comment]
@brian-f-harness the netlify link is broken/ not working for me? 😅

**Comment by brian-f-harness** [comment]
@Rohit-Kaliki The preview link in the conversation tab doesn't open for you?

https://658365d3e78d196952c17f86--harness-developer.netlify.app/docs/platform/role-based-access-control/manage-dashboards

File: docs/platform/role-based-access-control/manage-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+# Manage access control for dashboards
+
+This topic describes how to add and manage access control for dashboards.
+
+## Dashboards roles and permissions
+
+The following roles are required for dashboards:
+
+* **Dashboard - Static Editor**: To add, edit, and delete dashboards
```
Comment (Rohit-Kaliki) [inline docs/platform/role-based-access-control/manage-dashboards.md:15]: @brian-f-harness we have OOTB ACL Roles

https://app.harness.io/ng/account/6_vVHzo9Qeu9fXvj-AcbCQ/settings/access-control/roles/_dashboard_admin
https://app.harness.io/ng/account/6_vVHzo9Qeu9fXvj-AcbCQ/settings/access-control/roles/_dashboard_viewer

File: docs/platform/role-based-access-control/manage-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+To add and manage permissions for the **Dashboard - Static Editor** role, do the following:
+
+1. In **Harness**, select **Account Settings**, and then select **Access Control**.
+
+2. Select **Roles**.
+
+3. Select **New Role**. The New Role settings appear.
+
+4. In **Name**, enter **Dashboard - Static Editor** and select **Save**.
```
Comment (Rohit-Kaliki) [inline docs/platform/role-based-access-control/manage-dashboards.md:33]: We should reuse the OOTB roles for this section

File: docs/platform/role-based-access-control/manage-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+To add and manage permissions for the **Dashboard - All View** role, do the following:
+
+1. In **Harness**, select **Account Settings**, and then select **Access Control**.
+
+2. Select **Roles**.
+
+3. Select **New Role**. The New Role settings appear.
+
+4. In **Name**, enter **Dashboard - All View** and select **Save**.
```
Comment (Rohit-Kaliki) [inline docs/platform/role-based-access-control/manage-dashboards.md:55]: Same as above, let's leverage OOTB Roles

File: docs/platform/role-based-access-control/manage-dashboards.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+## Limit project access for STO dashboards
+
+:::info note
+
+Currently, this feature is behind the feature flag `CDB_PROJECT_RBAC`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+
+:::
+
+By default, RBAC for STO custom dashboards is restricted to entities exclusively. When you grant folder access to a user, they can view all entity information on the dashboard. Therefore, the **Project** filter on dashboards includes all STO projects across your organization by default, regardless of user RBAC project permissions.
```
Comment (Rohit-Kaliki) [inline docs/platform/role-based-access-control/manage-dashboards.md:75]: By default, RBAC for STO custom dashboards is restricted to + Dashboard + entities exclusively.

File: docs/platform/role-based-access-control/static/manage-access-control-for-dashboards-06.png
Comment (Rohit-Kaliki) [inline docs/platform/role-based-access-control/static/manage-access-control-for-dashboards-06.png:1]: Let's the the OOTB Roles here, sorry should have mentioned this in our call.

**Review by Rohit-Kaliki — CHANGES_REQUESTED** [review]

**Review by Rohit-Kaliki — APPROVED** [review]

File: docs/platform/role-based-access-control/manage-dashboards.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+sidebar_position: 80
+---
+
+# Manage access control for dashboards
+
+This topic describes how to add and manage access control for dashboards.
+
+## Dashboards roles and permissions
+
+The following roles are required for dashboards:
```
Comment (Rohit-Kaliki) [inline docs/platform/role-based-access-control/manage-dashboards.md:13]: Nit: We can also add the fact that the Account Admin/ Account Viewer Roles have this access baked in

**Review by Rohit-Kaliki — APPROVED** [review]

File: docs/platform/role-based-access-control/manage-dashboards.md (hunk 5 above)
Comment (brian-f-harness) [reply docs/platform/role-based-access-control/manage-dashboards.md:13]: added a note

**Pull request merged**